Makefile

```diff
@@ -1,42 +1,45 @@
-# Set the compiler, if it's not set by the environment.
-ifndef GXX
-	GXX = g++-9
+# If $CC and $CXX are defined as environment variables, those will be used here. However, if they aren't then GNU make
+# automatically defines them as `cc` and `g++`. Ultimately, we'd like those names to work, or the environment variables
+# to be set, but for the time being we need to override the defaults so that our existing dev environment works. This
+# can be removed when that is resolved.
+ifeq ($(CC),cc)
+CC = gcc-9
```

Comment:
I left these out on purpose after an earlier bug caused this to fail because a different line started with a tab, making it part of a recipe body, which isn't what was intended. I decided to just avoid that problem here.